templates/problem/list.html

```diff
@@ -232,6 +232,9 @@ <h3>{{ _('Hot problems') }} <i class="fa fa-fire"></i></h3>
                         <th class="ac-rate">
                             <a href="{{ sort_links.ac_rate }}">{{ _('AC %%') }}{{ sort_order.ac_rate }}</a>
                         </th>
+                        <th class="editorial">
+                            {{ _('Editorial') }}
```

Comment:
Doesn't this create the column even when in-contest?

Reply:
No, it's inside an `else`